### DISCUSSION TITLE
Bill of Materials

### DISCUSSION THREAD

**Discussion by nikandriko** [phord/Jarvis#13] [Q&A]
Hi @phord, wanted to validate with you the bill of materials (BoM) for this project:

- <s>[Digistump Oak](http://digistump.com/products/145) - not available</s>, alternative approach --> [ACEIRMC ESP8266 NodeMcu Mini D1 WeMos](https://www.amazon.com/gp/product/B081PX9YFV/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) 
- [2 x RJ45 8-Pin Connectors w/ Breakout boards](https://www.amazon.com/dp/B08HGS5G8N?psc=1&ref=ppx_yo2_dt_b_product_details)
- [1 x Adafruit Perma-Proto Half-sized Breadboard](https://www.amazon.com/gp/product/B00QSJ1T16/ref=ox_sc_act_title_1?smid=A2Q0FA2MED7CFH&psc=1)
- [{2 x 8-pin + 1 x 6-pin} male headers](https://www.amazon.com/Breakaway-Header-Straight-Connector-Prototype/dp/B07BXDYTBP/ref=sr_1_1?crid=14ERGC1W9AHIL&keywords=male+headers+break+off&qid=1643941552&sprefix=male+headers+break+off%2Caps%2C74&sr=8-1) - you can cut them off these long rows as needed
- <s>[1 x FTDI cable, aka USB-to-TTL serial cable ](https://www.amazon.com/gp/product/B06ZYPLFNB/ref=ox_sc_act_title_1?smid=A36ZSC50GWGFAQ&psc=1) for connecting PC to Digisrump Oak</s> - D1 Mini has a mini-USB connection, no need for UART.
- [1 x Jumper wires kit](https://www.amazon.com/gp/product/B07PMYTN43/ref=ox_sc_act_title_1?smid=AD98VXHC8N333&psc=1)
- [1 x Soldering iron](https://www.sparkfun.com/products/14456)
- [ 1 x spool of lead-free solder](https://www.sparkfun.com/products/9325) 

Am I missing anything else? Many thanks.

**Comment by phord** [comment]
You don't need the 6-pin header for the FTDI anymore.
The rest seems correct.

You may need a multimeter for troubleshooting.  This one is pretty decent: [Klein Tools MM300 Multimeter, Digital Voltmeter, AC/DC Voltage meter, Current, Resistance, Battery Tests, Diodes, and Continuity, 600V](https://www.amazon.com/Multimeter-Manual-Ranging-Klein-Tools-MM300/dp/B018O18VUW)

**Reply by phord** [reply]
I just realized you will also need one extra RJ-45 patch cable. Any old "straight" ethernet cable should be fine, such as this one: https://www.amazon.com/Monoprice-Cat6-Ethernet-Patch-Cable/dp/B003L1AET2

The length doesn't matter much. You may want to get a 3-foot or 5-foot one just because they are so ubiquitous.

**Comment by nikandriko** [comment]
This was my first version before ordering the printed boards that you provided Phil. 

![image](https://user-images.githubusercontent.com/962911/156494701-8986bcad-dcf7-4774-a0ed-c12da7aea749.png)
